code/modules/client/preference_setup/global/02_settings.dm

```diff
@@ -32,9 +32,9 @@
 				"asfx_togs",
 				"lastmotd" = "motd_hash",
 				"lastmemo" = "memo_hash",
-				"parallax_toggles" = "parallax_togs",
+				"parallax_toggles" = "toggles_secondary",
```

Comment:
Replace this line with `"toggles_secondary",` there is no need for the key-value pair, as both the column name and variable name will now line up.